api/proto/disperser/v2/disperser_v2.proto

```diff
@@ -19,6 +19,9 @@ service Disperser {
   
   // GetBlobCommitment is a utility method that calculates commitment for a blob payload.
   rpc GetBlobCommitment(BlobCommitmentRequest) returns (BlobCommitmentReply) {}
+
+	// GetPaymentState is a utility method to get the payment state of a given account.
+	rpc GetPaymentState(GetPaymentStateRequest) returns (GetPaymentStateReply) {}
```

Comment:
It would be more secure to ratelimit this endpoint, for now I just have it require authentication for the payment sender. Maybe we can add this in a subsequent PR? 
(unified the indent! It was the difference between a tab and two spaces that wasn't detected by my eyes or the IDE:p